### ISSUE TITLE
ace_text-input color control

### ISSUE THREAD

**Issue by guanw** [ajaxorg/ace#3772]
Hi, I'm using ace editor in our project for editing yaml files.

Our product use Oracle's OAT DevTools to provide accessibility feedback. In this particular component we have shown below, OAT tool is complaining about the color contrast of this grey cursor doesn't meet Oracle's requirement.

![image](https://user-images.githubusercontent.com/7311035/45646861-80946300-ba92-11e8-8244-cc1d3e2e107b.png)

Curious is there anyway to change the color of it? I tried to manipulate css color attribute but doesn't seem worked. Also tried to set the entire component `display: none` when the field is minimized and `display: inline-block` when maximized. But that approach seems to break functionality of ace itself.
Any suggestion is welcomed here! Thanks.

**Comment by guanw** [comment]
Forgot to mention, the element is <textarea class="ace_text-input "></textarea>

**Comment by nightwing** [comment]
the textarea  is a hidden element, that is not visible, the color for the visible cursor is https://github.com/ajaxorg/ace/blob/653005f8a3d8c1adeedd731d2d4f4feb1e5ca301/lib/ace/theme/textmate.css#L21

the color of cursor when blurred is defined by https://github.com/ajaxorg/ace/blob/653005f8a3d8c1adeedd731d2d4f4feb1e5ca301/lib/ace/css/editor.css#L297

> Also tried to set the entire component display: none

If you change the size or visibility of the editor, you need to call `editor.resize()` to let editor know that it needs to redraw.

**Comment by guanw** [comment]
@nightwing Hi, I tried overwriting the cursor opacity to 1. But the tool still complains. 
![image](https://user-images.githubusercontent.com/7311035/45765236-8f9a2300-bc02-11e8-9775-db8ffb2b69fc.png)

Looking at the warning rule, it's complaining about `ace_text-input` and that background color couldn't be determined because the some other element overlapped with this ace_text-input element?

And for your second suggestion about editor.resize(), we have that method called everytime the editor is resized to minimum or maximum width/height, but i think after first time `display` of ace_text-input is set to `none`, it seems event like focus/blur will be blocked forever.

Any other suggestions? If not, I will probably marked this as not feasible to fix.

**Comment by nightwing** [comment]
I thought you were hiding the editor itself, changing any styles on ace_text-input itself will break the editor.

As for the warning it is shown because the real textarea is completely hidden, and cursor is another div, which the tool can not detect.

**Comment by guanw** [comment]
Got you. I'll mark it as not feasible to fix then. Thanks.
